.github/workflows/dockerhub-description.yml

```
@@ -0,0 +1,21 @@
+on:
+  push:
+    branches:
+      - master
```

I've renamed the main branch to `main`
```suggestion
      - main
```